rosa_tutorials/rosa-mobb-azure-groupsync.adoc

```diff
@@ -0,0 +1,176 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-azure-groupsync"]
+= Tutorial: Using Group Sync Operator with Azure Active Directory and ROSA
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-azure-groupsync
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-18
+// ---
+// date: '2021-11-08'
+// title: Using Group Sync Operator with Azure Active Directory and ROSA
+// tags: ["Azure", "ROSA"]
+// authors:
+//   - Steve Mirman
+//   - Kumudu Herath
+// ---
+
+include::snippets/mobb-support-statement.adoc[leveloffset=+1]
+
+This guide focuses on how to synchronize Identity Provider (IDP) groups and users after configuring authentication in OpenShift Cluster Manager (OCM). For an IDP configuration example, please reference the link:https://mobb.ninja/docs/idp/azuread[Configure Azure AD as an OIDC identity provider for ROSA/OSD] guide.
+
+// ALLOW PR#64888 TO MERGE AND REPLACE THIS URL WITH: xref:../rosa_tutorials/rosa-mobb-azure-oidc[Configure Azure AD as an OIDC identity provider for ROSA/OSD] guide.
+
+To set up group synchronization from Azure Active Directory (AD) to ROSA/OSD, you must:
+
+. Define groups and assign users in Azure AD
+. Add the required API permissions to the app registration in Azure AD
+. Install the Group Sync Operator from the OpenShift Operator Hub
+. Create and configure a new Group Sync instance
+. Set a synchronization schedule
+. Test the synchronization process
+
+[id="rosa-mobb-azure-groupsync-define-groups"]
+== Define groups and assign users in Azure AD
+
+[NOTE]
+====
+To synchronize groups and users with ROSA/OSD, they must exist in Azure AD.
+====
+
+. Create groups to synchronize with ROSA/OSD if they do not already exist:
++
+image:az-ad-grp.png[Azure AD Groups]
+
+. Create user IDs to synchronize with ROSA/OSD if they do not already exist:
++
+image:az-ad-usr.png[Azure AD Users]
+
+. Assign newly created users to the appropriate group:
++
+image:az-ad-assign.png[Azure AD add user to group]
+
+[id="rosa-mobb-azure-groupsync-add-api-permissions"]
+== Add API Permissions to Azure AD App Registration
+
+The GroupSync job requires permissions on the Azure AD tenant beyond those of the OIDC IdP. For it to work, add the following entries under the 'API Permissions' menu item:
+
+* `Group.Read.All`
+* `GroupMember.Read.All`
+* `User.Read.All`
+
+These three permissions should all be 'Application' rather than 'Delegated' and this will require clicking on 'Grant admin consent' button above the permissions list. When done, the screen should look like this:
```

Comment:
```suggestion
These three permissions should all be `Application` rather than `Delegated` and this will require clicking the *Grant admin consent* button above the permissions list. When done, the screen should look like this:
```